src/migrator/migrator.ts

```diff
@@ -21,6 +22,25 @@ export type Migration = string | MigrationFn
 
 export class MigrationError extends TypedError {}
 
+export const postRun = (tx: Tx, model: Model): Promise<void> => {
+	const { initScript } = model
+	if (initScript == null) {
+		return Promise.resolve()
+	}
+
+	const modelName = model.apiRoot
+
+	return Promise.using(lockMigrations(tx, modelName), () =>
+		checkModelAlreadyExists(tx, modelName)
+		.then((exists) => {
+			if (!exists) {
+				sbvrUtils.api.migrations.logger.info('First time executing, running init script')
+				return tx.executeSql(initScript).return()
```

Comment:
`initScript` makes you expect you can specify the path to a shell script -- it would probably be better to name the attribute `initSql` instead.

Reply:
done